src/main/java/one/util/streamex/PrefixOps.java

```diff
@@ -299,4 +299,224 @@ public void accept(T next) {
             }
         }
     }
+    
+    static final class OfUnordInt extends PrefixOps<Integer, Spliterator.OfInt> implements IntConsumer, Spliterator.OfInt {
+        private final LongBinaryOperator op;
+        private final IntBinaryOperator localOp;
+        private boolean started;
+        private AtomicLong accRef;
+        private int acc;
+        
+        OfUnordInt(Spliterator.OfInt source, IntBinaryOperator op) {
+            super(source);
+            this.localOp = op;
+            this.op = (a, b) -> a == Long.MAX_VALUE ? b : op.applyAsInt((int) a, (int) b);;

```

Comment:
>I'd prefer Math.toIntExact(a) instead of (int) a to be able to manifest a possible error.

I believe that kind of error is impossible here: all long operands here are basically int operands previously upcasted to long.

> Also, extra semicolon.
Fixed